server/src/main/java/com/objectcomputing/checkins/services/checkins/CheckInController.java

```diff
@@ -33,10 +33,14 @@
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
 @Tag(name="check-ins")
+@Singleton
```

Comment:
Anything annotated with @Controller can also be treated as a Singleton so using both annotations is unnecessary.